configs/components/oifs/oifs.yaml

```diff
@@ -653,8 +655,8 @@ outdata_sources:
         ICMUA: ${out_date_folder}/ICMUA${oifs.input_expid}+*
         MPP: ${out_date_folder}/MPP*
         # XIOS output
-        oifsnc: ${out_date_folder}/${input_expid}_*_${start_date!syear!smonth!sday}_*.nc
-        awi_cm3: ${out_date_folder}/awi_cm3*
+        oifsnc: ${input_expid}_*_${start_date!syear!smonth!sday}_*.nc
+        atm: atm*
```

Comment:
@joakimkjellsson , @seb-wahl , are you okay with these changes? The basically imply that, for the XIOS output, the files don't end up in time-stamped folders, but instead, directly in the `outdata/oifs` folder.

Reply:
Makes working with e.g. cdo easier.

Reply:
I think that makes a lot more sense. So I approve!